app/javascript/dashboard/routes/dashboard/settings/reports/components/SLA/SLAReportFilters.vue

```diff
@@ -0,0 +1,91 @@
+<template>
```

Comment:
Used the the current date picker. We will change to new design later.

Reply:
Cool, let's try to ensure we use `v-model` instead of change events for the new component

CC: @iamsivin